span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGenerator.java

```diff
@@ -450,6 +452,39 @@ private static void setUrl(
         .ifPresent(url -> httpBuilder.getRequestBuilder().setUrl(url));
   }
 
+  private static void setScheme(
+      Http.Builder httpBuilder, Map<String, JaegerSpanInternalModel.KeyValue> tagsMap) {
+    Optional<String> optionalScheme =
+        FirstMatchingKeyFinder.getStringValueByFirstMatchingKey(
+            tagsMap, List.of(HTTP_REQUEST_FORWARDED), s -> !StringUtils.isBlank(s));
```

Comment:
HTTP_REQUEST_FORWARDED, and HTTP_REQUEST_X_FORWARDED_PROTO has higher precedence over HTTP_SCHEME of otel semantics?

Reply:
yes